src/KeyManager.ts

```diff
@@ -5,8 +5,7 @@ import { promisify } from 'util'
 
 // js imports
 const kbpgp = require('kbpgp')
```

Comment:
Why are we not using typescript `import ...`?

Reply:
These js-imports have been changed to import statements